src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -496,26 +526,23 @@ LRESULT FancyZones::WndProc(HWND window, UINT message, WPARAM wparam, LPARAM lpa
 {
     switch (message)
     {
-    case WM_HOTKEY:
-    {
+    case WM_HOTKEY: {

```

Comment:
Same here, the older format is more readable and more consistent with how curly braces are used in other statements.

Reply:
done.